File: 3396-valid-word/3396-valid-word.java
class Solution {
    public boolean isValid(String word) {
       if(word.length()<3){
        return false;
       } 
       int V=0;
       int C=0;
       for(char c: word.toCharArray()){
        if(Character.isLetter(c)){
            if("aeiouAEIOU".indexOf(c)!=-1) V+=1;
            else C+=1;
        }
        else if(!Character.isDigit(c)){
            return false;
        }
       }
       return V>=1 && C>=1;
    }
}

File: 3396-valid-word/3396-valid-word.py
class Solution:
    def isValid(self, word: str) -> bool:
        if(len(word)<3):
            return False
        vowels=0
        consonents=0
        v_set="aeiouAEIOU"
        for c in word:
            if c.isalpha():
                if c in v_set:
                    vowels+=1
                else:
                    consonents+=1
            elif not c.isdigit():
                return False
        return vowels>=1 and consonents>=1

